cranelift/codegen/src/isa/x86/unwind.rs

```diff
@@ -215,11 +335,37 @@ impl UnwindInfo {
             return None;
         }
 
+        if static_frame_allocation_size > 240 && saved_fpr {
+            panic!("stack frame is too large to use with Windows x64 SEH when preserving FPRs");
+        }
+        if static_frame_allocation_size % 16 != 0 {
+            eprintln!("static frame allocation size: {}", static_frame_allocation_size);
+            panic!("bad frame size");
+        } else {
+            assert!(
+                static_frame_allocation_size % 16 == 0,
```

Comment:
This is always true when reached.

Reply:
Yes, this is back to being somewhat wip - wasmtime yielded some new errors I hadn't seen testing only in cranelift. I'd tried setting up a build environment on the Windows machine I have available but ran afoul of link.exe being... wrong? So I just added a bit of logging in the hopes I could figure out what went wrong from just a bit of CI logs. I think I have, and once it's passing I'm going to revert `29c6c40 `.

Reply:
(I've reverted `29c6c40` and the assertion this comment is on is once again a meaningful)